cms/static/cms/js/widgets/forms.pageselectwidget.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright https://github.com/divio/django-cms
+ */
+
+// #############################################################################
+// NAMESPACES
+var django = window.django || {};
+
+// #############################################################################
+// PAGE SELECT WIDGET
+// cms/forms/widgets.py
+(function ($) {
+    'use strict';
+
+    // shorthand for jQuery(document).ready();
+    $(function () {
+
+        var options = window._PageSelectWidget;
+        var group0 = $('#id_' + options.name + '_0');
+        var group1 = $('#id_' + options.name + '_1');
+        var group2 = $('#id_' + options.name + '_2');
+        var tmp;
+
+        // handles the change event on the first select "site"
+        // that controls the display of the second select "pagetree"
+        group0.on('change', function () {
+            tmp = $(this).children(':selected').text();
+
+            group1.find('optgroup').remove();
+            group1.append(
+                group2.find('optgroup[label="' + tmp + '"]').clone()
+            ).change();
+
+            // refresh second select
+            setTimeout(function () {
+                group1.trigger('change');
+            }, 0);
+        }).trigger('change');
+
+        // sets the correct value
+        group1.on('change', function () {
+            tmp = $(this).find('option:selected').val();
+
+            if (tmp) {
+                group2.find('option').attr('selected', false);
```

Comment:
strictly speaking it should be `removeAttr('selected')` or `prop('selected', false)` but i don't know if it makes sense to change things now